.github/ISSUE_TEMPLATE/code_formatting_guidelines_specific_issue.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Formatting or code style guidelines issue
+about: Discuss editor / tooling formatting concerns and code style guidelines
+title: ''
+labels: [style-guide, under-discussion]
+assignees: '@nojaf,@KathleenDollard,@dsyme'
+---
+
+Depending the concern being discussed, erase the other top sections and review/tick what applies the most.
+
+# Is your request about a feature in code formatting (using fantomas, .editorconfig, editors with formatting options)?
+
+Formatting infrastructure decision checklist:
+* [ ] If this is a code formatter discussion, please suggest a detailed list of settings names and their default values, with their behaviour
+* [ ] the choice for tooling support should be based on ease of implementation in the code formatter
+* [ ] the choice for tooling output should be based on
+  * [ ] conserving original formatting but for specific predicate that makes code significantly less maintainable
+  * [ ] maximizing uniformity of a specific construct across a particular codebase
+* [ ] I would like to contribute testing the feature implementation as end user
+* [ ] I would like to contribute unit tests so the tooling implementation covers my use cases
+* [ ] I would like to contribute implementation
+  * [ ] pro-bono
+  * [ ] sponsored
+  * [ ] sponsoring
+* [ ] The implementation requires the several options upfront (rather than just the most "agreed upon") [^1]
+* [ ] This is something that is common place in other langauges (add items nested under this list with one language per line)
+* [ ] There are examples of codeformatters settings in other languages / tooling available (add items nested under this list with one link per line), add screenshots to the description
+* [ ] There are examples of codeformatters implementation available (add items nested under this list with one link per line)
+
+[^1] this is generally evaluated by the person implementing and the entity potentially sponsoring the work
```

Comment:
```suggestion
Formatting infrastructure decision checklist:
* [ ] I would like to contribute testing the feature implementation as end user
* [ ] I would like to contribute unit tests so the tooling implementation covers my use cases
* [ ] I would like to contribute implementation
  * [ ] pro-bono
  * [ ] sponsored
  * [ ] sponsoring

```


Removing rest of the checklist will make it easier to accept this as a template.
The points are too specific and in many cases not relevant.
(opposite does not imply - even when whey are missing in a template, they can be brought up individually if it makes sense)

Reply:
@T-Gro makes sense, I think the context when I came up with proposal for template was:
* discussions that implied that code formatting guidelines would be tied to fact we like to have formatting tool that can potentially makes the suggestion automatic
* sense on my part, that if many such discussions come, it is good to have the recurrent questions that will eventually come addressed upfront
* inviting to sponsor implementation in the formatter when it makes sense

I kind of sumarized what I inferred was the process based on long exchanges on a formatting suggestion.